hack/dpu-mode.yaml

```diff
@@ -2,8 +2,10 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-    name: dpu-mode-config
+    name: smart-nic-dpu-mode-config
```

Comment:
This is not just about mode now given the resource-name is added,  maybe use smart-nic-dpu-config or hardware-offload-config?

Reply:
renamed to ```hardware-offload-config```. Thanks for catching this.